### DISCUSSION TITLE
Why pyodide loading time is increasing with newer versions?

### DISCUSSION THREAD

**Discussion by albertas** [pyodide/pyodide#4242] [General]
I have observed, that pure pyodide loading time has increased significantly in the latter versions. My benchmarking showed these loading times:
- ~2.694 seconds for pyodide v0.21.3
- ~2.645 seconds for pyodide v0.22.1
- ~4.312 seconds for pyodide v0.23.4
- ~4.401 seconds for pyodide v0.24.1 

Is it expected?

**Comment by hoodmane** [comment]
Well we have a size benchmark but we should probably add a loading time one. But most likely what this shows is that Python 3.11 added a lot of loading time, since 0.21 and 0.22 are about the same and 0.23 and 0.24 are about the same and 0.22 to 0.23 updated the Python version. It would be great to look into this more though. Also, I wonder what happens with Python 3.12...
